easybuild/easyconfigs/e/Extrae/Extrae-4.1.6-gompi-2023b.eb

```
@@ -0,0 +1,35 @@
+name = 'Extrae'
+version = '4.1.6'
+
+homepage = 'https://tools.bsc.es/extrae'
+description = """Extrae is the package devoted to generate Paraver trace-files for a post-mortem analysis. 
+Extrae is a tool that uses different interposition mechanisms to inject probes into the target application 
+so as to gather information regarding the application performance."""
+
+toolchain = {'name': 'gompi', 'version': '2023b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://ftp.tools.bsc.es/%(namelower)s']
+sources = ['%(namelower)s-%(version)s-src.tar.bz2']
+patches = ['Extrae-4.1.6-detect_binutils.patch']
+checksums = [
+    # extrae-4.1.6-src.tar.bz2
+    ('9f146e70311b8ae9d77584f6efc7b30478885cfd095f7bd3937d5b08aec19985',
+     'f96b2c3586a0e83c6fa08d3136a8f399715969b5bb7ee6792c3fd93667d495fd'),
+    '7c4d87d12a5383ccdfdd75967a207b9d9aca27c707c499cd6ef0a27fd4989483',  # Extrae-4.1.6-detect_binutils.patch
+]
+
+builddependencies = [
+    ('Automake', '1.16.5'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13', '', ('GCCcore', '13.2.0')),
+    ('Boost', '1.83.0', '', ('GCC', '13.2.0')),
+    ('libunwind', '1.6.2', '', ('GCCcore', '13.2.0')),
+    ('libxml2', '2.11.5', '', ('GCCcore', '13.2.0')),
+    ('libdwarf', '0.9.2', '', ('GCCcore', '13.2.0')),
+    ('PAPI', '7.1.0', '', ('GCCcore', '13.2.0')),
+]
+
```

```suggestion

runtest = 'check'

```
You're currently running the test suite